app/views/spree/shared/_products.html.erb

```diff
@@ -0,0 +1,28 @@
+<%
+  paginated_products = @searcher.retrieve_products if params.key?(:keywords)
+  paginated_products ||= products
+%>
+
+<% content_for :head do %>
+  <% if paginated_products.respond_to?(:total_pages) %>
+    <%= rel_next_prev_link_tags paginated_products %>
+  <% end %>
+<% end %>
+
+<% if params.key?(:keywords) %>
```

Comment:
Here we intentionally removed the no_products_found message?
https://github.com/solidusio/solidus/blob/master/frontend/app/views/spree/shared/_products.html.erb#L13

Reply:
Yes, it was an error in the logic of the old frontend, a part of the code was never called.